azuredevops/resource_repository.go

```diff
@@ -22,35 +22,40 @@ import (
 	"github.com/microsoft/azure-devops-go-api/azuredevops/v6/git"
 )
 
-var _ gitprovider.UserRepository = &userRepository{}
+var _ gitprovider.OrgRepository = &repository{}
 
-func newUserRepository(ctx *clientContext, apiObj git.GitRepository, ref gitprovider.RepositoryRef) *userRepository {
-	return &userRepository{
+func newRepository(ctx *clientContext, apiObj git.GitRepository, ref gitprovider.RepositoryRef) *repository {
+	return &repository{
 		clientContext: ctx,
 		r:             apiObj,
 		ref:           ref,
 		pullRequests: &PullRequestClient{
 			clientContext: ctx,
 			ref:           ref,
 		},
+		trees: &TreeClient{
+			clientContext: ctx,
+			ref:           ref,
+		},
 	}
 }
 
-type userRepository struct {
+type repository struct {
```

Comment:
since within Azure Devops it doesn't have a concept of user repositories vs org repositories